mmv1/products/networkservices/product.yaml

```diff
@@ -16,7 +16,7 @@ name: NetworkServices
 versions:
   - !ruby/object:Api::Product::Version
     name: beta
-    base_url: https://networkservices.googleapis.com/v1/
+    base_url: https://networkservices.googleapis.com/v1beta1/
```

Comment:
Given the test failures, I'm specifically concerned about this change. Have you tested locally? Are we sure this is not a breaking change?

Reply:
@roaks3 I ran some tests again in my local environment and this "beta1" made a few tests fail. I already made a push with the code removing it.

But I saw that most of the tests were failing because of those 2 errors:
```
Error: Error creating EdgeCacheKeyset: googleapi: Error 404: Method not found.
Error: Error creating EdgeCacheOrigin: googleapi: Error 404: Method not found.
```
I don't have access to the logs. Can you validate for me if this is the case?

Reply:
Yea, re-running the tests now, but an example error was:

`The requested URL /v1beta1/projects/ci-test-project-188019/locations/global/edgeCacheKeysets?alt=json&amp;edgeCacheKeysetId=tf-test-my-keysetn946pboxbs was not found on this server`

which is why I was wondering about the v1 -> v1beta1 change.

Reply:
Weird, for some reason that fixed the tests here on the PR.
But my local environment was still failing with the messages I sent.

Well, thanks for re-running!
And now that the test passed you can do the full review.